src/twisted/trial/_dist/test/test_worker.py

```diff
@@ -22,7 +22,8 @@
 from twisted.internet.defer import fail, succeed
 from twisted.internet.main import CONNECTION_DONE
 from twisted.internet.error import ConnectionDone
-from twisted.python.compat import _PY3, NativeStringIO as StringIO, unicode
+from twisted.python.compat import (NativeStringIO as StringIO, unicode)
```

Comment:
`()`s on a single-line import look odd.  Not sure if this is following some more general pattern.  I'd probably leave them off, if I were making the change.